examples/tokio-streams.rs

```diff
@@ -0,0 +1,36 @@
+extern crate futures;
+extern crate inotify;
+extern crate tempdir;
+extern crate tokio;
+
+use inotify::{Inotify, WatchMask};
+use std::{fs::File, io, thread, time::Duration};
+use tempdir::TempDir;
+use tokio::prelude::*;
+
+/// XXX: hic sunt dracones
+static mut BUFFER: [u8; 32] = [0; 32];
```

Comment:
You can move the static into the function. Still bad, as it still requires unsafe, but at least it won't be accessible from anywhere else.

Reply:
Could you also add a comment explaining that this is a workaround for #120, and should be avoided, if possible?